### ISSUE TITLE
Installation Problems

### ISSUE THREAD

**Issue by thenewt15** [davodesign84/react-native-mixpanel#120]
My Mixpanel object is undefined after trying to import it.

`import Mixpanel from 'react-native-mixpanel';`
\- OR -
`const Mixpanel = require('react-native-mixpanel');`

I follow the instructions pretty closely:
- `yarn add react-native-mixpanel`
- `react-native link react-native-mixpanel`
- Moved into `ios` folder and did `pod init`
- Tried both `pod Mixpanel` and `pod Mixpanel-swift` in the podfile. Neither worked.
- Ran `pod install`. Confirmed the pod was added to the XCode workspace.

Any calls to Mixpanel functions crash the app because I'm trying to call a function on an undefined object. So this would crash the app.
`Mixpanel.sharedInstanceWithToken(YOUR_PROJECT_TOKEN);`

How can I go about debugging this problem? Any help would be appreciated.

**Comment by Maddoc42** [comment]
Hey @thenewt15. We are seeing a similar problem here. What version of React Native are you using?

Previously we installed the RN Firebase package also via Pods which seems to have a problem with RN 0.50.0 --> https://github.com/invertase/react-native-firebase/issues/636. The solution offered by @chrisbianca helped us with Firebase. Hopefully this helps!

**Comment by Maddoc42** [comment]
@thenewt15 can confirm that this was the issue in our case. Here are the steps we took:

```
react-native unlink react-native-mixpanel
rm node_modules/react-native-mixpanel/react-native-mixpanel.podspec
react-native link react-native-mixpanel
```

Again, we are using RN 0.50.0.

**Comment by thenewt15** [comment]
Ok got it working.
@Maddoc42 I gave your suggestion a shot but it didn't work in my case.

Something that helped me debug it was just going into XCode and trying to build. It failed with a linker error saying `ld: library not found for -lRNMixpanel-tvOS`.

I had previously removed the tvOS schemes that came with my Xcode project, because I didn't need them. So I figured I didn't need the tvOS Mixpanel library either. To fix, I removed `libRNMixpanel-tvOS.a` from my 'Link Binary With Libraries' section of my project settings. After that, everything is working.
